hack/test-e2e.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -x
+set -e
+
+ARTIFACT_DIR=${ARTIFACT_DIR:-_output}
+manifest=$(mktemp)
+global_manifest=$(mktemp)
+
+cleanup(){
+  local return_code="$?"
+  set +e
+
+  cat $manifest > $ARTIFACT_DIR/manifest
+  cat $global_manifest > $ARTIFACT_DIR/global_manifest
+
+  exit $return_code
+}
+trap cleanup exit
+
+if [ -n "${IMAGE_FORMAT:-}" ] ; then
```

Comment:
why `${xxx:-}`? Is it the same as plain `"${xxx}"`?

Reply:
fixed.